api/rest/coordinator_test.go

```diff
@@ -110,9 +112,10 @@ func TestCoordinatorCreateAndCollect(t *testing.T) {
 		client.On("GetFile", ctx, testData.n.baseURL, localBundleID, testData.zipPath).Return(nil)
 	}
 
-	statuses := c.CreateBundle(ctx, localBundleID, []node{node1, node2, node3})
+	report := newBundleReport("cluster-bundle")
+	statuses := c.CreateBundle(ctx, localBundleID, []node{node1, node2, node3}, report)
```

Comment:
I do not like this approach. Reports should be returned by Coordinator functions and merged at the end in to a single report. With approach like this we have a shared state object in highly parallel world. This may not be easy to handle.

Reply:
@makkes @drewvanstone What do you think?